PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -355,6 +366,72 @@ RUN=`opts_GetOpt1 "--printcom" $@`
 UseJacobian="$(echo ${UseJacobian} | tr '[:upper:]' '[:lower:]')"
 UseJacobian=`opts_DefaultOpt $UseJacobian "true"`
 
+
+# ------------------------------------------------------------------------------
+#  Check MMP Version
+# ------------------------------------------------------------------------------
+
+Compliance="hcp"
+MPPVersion=`opts_GetOpt1 "--mppversion" $@`
+MPPVersion=`opts_DefaultOpt $MPPVersion "hcp"`
+
+if [ "${MPPVersion}" = "legacy" ] ; then
+  log_Msg "Legacy Minimal Preprocessing Pipelines: PreFreeSurferPipeline v.XX"
```

Comment:
Again, "Legacy Minimal Preprocessing Pipelines" strongly implies legacy _code_ to me.  Better to frame this as "Minimal Preprocessing Pipeline adaptations to support legacy-style acquisitions".

Reply:
MPP supports HCP-Style and Legacy-Style acquisitions.